packages/office-ui-fabric-react/src/components/Link/Link.styles.ts

```diff
@@ -25,7 +25,16 @@ export const getStyles = (props: ILinkStyleProps): ILinkStyles => {
           [`.${IsFocusVisibleClassName} &:focus`]: {
             // Can't use getFocusStyle because it doesn't support wrapping links
             // https://github.com/OfficeDev/office-ui-fabric-react/issues/4883#issuecomment-406743543
-            outline: `1px solid ${theme.palette.neutralSecondary}`
+            // A box-shadow allows the focus rect to wrap links that span multiple lines
+            // and helps the focus rect avoid getting clipped.
+            boxShadow: `0 0 0 1px ${theme.palette.neutralSecondary} inset`
+          },
+          '@media screen and (-ms-high-contrast: active)': {
+            // For IE high contrast mode
+            borderBottom: 'none',
+            selectors: {
+              '&:focus': { outline: '1px solid WindowText' }
```

Comment:
instead of nesting this, you should have a separate rule:

```
'@media screen and (-ms-high-contrast: active) &:hover': {
}
```

And I believe there is a constant you can use for the hc media query.

Reply:
https://codepen.io/dzearing/pen/daXjxM

Reply:
I tried to do this separately but it causes the outline to not be visible in HC mode...

Reply:
http://fabricweb.z5.web.core.windows.net/pr-deploy-site/refs/pull/7793/merge/fabric-website-resources/dist/index.html#/examples/link - just double checked outline doesn't show in HC mode :(